src/cascadia/TerminalSettingsModel/VcDevCmdGenerator.h

```diff
@@ -0,0 +1,46 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- VcDevCmdGenerator
+
+Abstract:
+- Dynamic profile generator for Visual C++ development environments.
+
+Author(s):
+- Heath Stewart - September 2021
+
+--*/
+
+#pragma once
+#include "VisualStudioGenerator.h"
+#include "VsSetupConfiguration.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Model
+{
+    class VcDevCmdGenerator final : public VisualStudioGenerator::IVisualStudioProfileGenerator
+    {
+    public:
+        void GenerateProfiles(const VsSetupConfiguration::VsSetupInstance& instance, bool hidden, std::vector<winrt::com_ptr<implementation::Profile>>& profiles) const override;
+
+    private:
+        bool IsInstanceValid(const VsSetupConfiguration::VsSetupInstance&) const
+        {
+            // TODO: Need to determine when these VC environment scripts were added. vcvars32.bat at least was there since 15.0.

```

Comment:
"TODO eventually". I'm working to find out, but how the code is written if those files aren't there, there'd be no error. I'll find out ASAP. If you want to merge this PR otherwise, I can follow it up with another as soon as I close on a decision with the VC team.

Reply:
I was able to track down the Dev 15.0 (latest service version) and indeed the path was the same then as it is in VS2019 (latest) and VS2022 Preview (latest). This path should be fine, then. I'll remove the TODO.